README.md

```diff
@@ -43,6 +43,18 @@ If libroc is not in default path you can specify `ROC_INCLUDE_PATH` (path to roc
 
 Additional compilation and linking flags can be specified respectively with `CFLAGS` and `LDFLAGS` gradle system variables
 
+#### Android Release
+
+Release workflow:
+ * make github release with tag version `v-x.x.x`, e.g. `v-0.0.1`
```

Comment:
Nitpick: do we need this dash? The usual convention is v1.2.3 instead of v-1.2.3. In roc-toolkit repo we don't have the dash too.

Reply:
Unfortunately, the plugin only support prefixes which matches `[/@]?([A-Za-z]+[/@-])+`
So, we could use prefixes, like:
- `release/0.0.1`
- `version@0.0.1`
- `v-0.0.1`

Another approach: set version without using a plugin, like this
```
./gradlew build -Pversion=0.0.1
```

Reply:
That's odd: https://github.com/palantir/gradle-git-version/pull/61#discussion_r124338382

Not very critical, but I'd prefer to use the same version naming across the toolkit and its bindings, to avoid additional source of confusion. So if possible maybe send them a PR or switch to another plugin?

> Another approach: set version without using a plugin, like this

Or maybe just call `git describe --tags` by hand?

Reply:
I also don't like this dash very much :)
I'll fix this